assets/i18n/en/user.arb

```diff
@@ -911,7 +911,7 @@
     "type": "text",
     "placeholders": {}
   },
-  "clear_app_preferences_desc": "Clear the application preferences. Currently this is only the preferred order of comments.",
+  "clear_app_preferences_desc": "Clear the application preferences. Currently only the preferred order of comments and the trusted domains.",
   "@clear_app_preferences_desc": {
```

Comment:
Minor English correction, this sentence should be:
"Clear application preferences. Currently only the preferred order of comments and trusted domains"

Reply:
To me that reads more like "the preferred order of [comments and trusted domains]", i.e. as if the order of the trusted domains would be cleared. 🤔

Reply:
Ah  ok makes sense, then we should perhaps say `preferred order of comments and the list of trusted domains.`

Reply:
Yeah, that sounds good to me. 👍